bob-apps/bin/config_cluster_generator/utils.rs

```diff
@@ -83,21 +83,10 @@ fn deserialize(content: String) -> AnyResult<ClusterConfig> {
         .map_err(|e| anyhow!("deserialize: ERR [{}]", e))
 }
 
-/// greatest common divider
-pub fn gcd(a: usize, b: usize) -> usize {
-    debug!("gcd of {} and {}", a, b);
-    if a == 0 {
-        b
-    } else if b == 0 {
-        a
+pub fn ceil(a: usize, b: usize) -> usize {
```

Comment:
https://doc.rust-lang.org/std/primitive.f64.html#method.ceil

Reply:
Ceil for usize is available only on nightly https://doc.rust-lang.org/std/primitive.usize.html#method.div_ceil. Or should I covert usize to f64, calculate ceil and then convert result back to usize?

Reply:
Didn't notice that you are working with `usize` only. The you can keep this code